code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -83,6 +83,12 @@
 	icon_state = "bottle17"
 	list_reagents = list("charcoal" = 30)
 
+/obj/item/reagent_containers/glass/bottle/ether
```

Comment:
It's my educated decision to say that ether in bottles is useless and shouldn't be put into vending machines, which you didn't do.

Since it's unobtainable, we might as well remove it.

👌

Reply:
`Educated`